sample-operators/leader-election/src/main/java/io/javaoperatorsdk/operator/sample/LeaderElectionTestReconciler.java

```diff
@@ -2,15 +2,14 @@
 
 import java.time.Duration;
 import java.util.ArrayList;
+import javaoperatorsdk.sample.v1.LeaderElection;
+import javaoperatorsdk.sample.v1.LeaderElectionStatus;
```

Comment:
These two classes should be put in the proper package but that can be done in a different PR.